src/python/pants/core/goals/fmt.py

```diff
@@ -286,7 +286,8 @@ async def fmt_language(language_fmt_request: _LanguageFmtRequest) -> _LanguageFm
             continue
         result = await Get(FmtResult, FmtRequest, request)
         results.append(result)
-        prior_formatter_result = result.output
+        if not result.skipped:
+            prior_formatter_result = result.output
```

Comment:
Question for @Eric-Arellano @stuhood 

Is the intention for skipped formatters to return an empty digest or just forward through the original input digest (untouched)? 

As in, is the core intention for formatters to be pipelined (in some deterministic order), or is this a workaround/side-effect because skipped formatters happen to have empty digests?

Reply:
It's a good point. Right now "skipped" is a magical combo of values. What would it look like if it was distinct?

Reply:
Yeah I don't think it matters much how Skip is factored, as long we don't waste time trying to run it etc